### PR TITLE
Add Lima

### PR DESCRIPTION
**Pull request by AkihiroSuda**
Lima was approved as a CNCF Sandbox project recently
- https://github.com/cncf/toc/issues/924

cc @jandubois @mattfarina

### Pre-submission checklist:

*Please check each of these after submitting your pull request:*

* [X] Are you only including a `repo_url` if your project is 100% open source? If so, you need to pick the single best GitHub repository for your project, not a GitHub organization.
* [X] Is your project closed source or, if it is open source, does your project have at least 300 GitHub stars?
* [X] Have you picked the single best (existing) category for your project?
* [X] Does it follow the other guidelines from the [new entries](https://github.com/cncf/landscape#new-entries) section?
* [X] Have you added your SVG to `hosted_logos` and referenced it there?
* [X] Does your logo clearly state the name of the project/product and follow the other logo [guidelines](https://github.com/cncf/landscape#logos)?
* [X] Does your project/product name match the text on the logo?
* [X] Have you verified that the Crunchbase data for your organization is correct (including headquarters and LinkedIn)?
* [ ] ~30 minutes after opening the pull request, the CNCF-Bot will post the URL for your staging server. Have you confirmed that it looks good to you and then added a comment to the PR saying "LGTM"?

### CODE REVIEW

File: landscape.yml [hunk 1]
```diff
@@ -1800,6 +1800,19 @@ landscape:
...
+            project: sandbox
+            repo_url: https://github.com/lima-vm/lima
+            logo: lima.svg
+            crunchbase: https://www.crunchbase.com/organization/cloud-native-computing-foundation
+            extra:
+              accepted: '2022-09-13'
+              dev_stats_url: https://lima.devstats.cncf.io/
+              artwork_url: https://github.com/cncf/artwork/blob/master/examples/sandbox.md#lima-logos
+              slack_url: https://slack.cncf.io/
+              chat_channel: '#lima'
```
Comment (AkihiroSuda) [inline landscape.yml:1815]: (This channel is not created yet)

File: landscape.yml [hunk 2]
```diff
@@ -1800,6 +1800,19 @@ landscape:
...
+            name: Lima
+            description: Linux virtual machines, typically on macOS, for running containerd
+            project: sandbox
+            repo_url: https://github.com/lima-vm/lima
+            logo: lima.svg
+            crunchbase: https://www.crunchbase.com/organization/cloud-native-computing-foundation
+            extra:
+              accepted: '2022-09-13'
+              dev_stats_url: https://lima.devstats.cncf.io/
+              artwork_url: https://github.com/cncf/artwork/blob/master/examples/sandbox.md#lima-logos
```
Comment (AkihiroSuda) [inline landscape.yml:1813]: (Being added in https://github.com/cncf/artwork/pull/361)

File: landscape.yml [hunk 3]
```diff
@@ -1800,6 +1800,19 @@ landscape:
               accepted: '2021-04-28'
               dev_stats_url: https://wasmedge.devstats.cncf.io/
               artwork_url: https://github.com/cncf/artwork/blob/master/examples/sandbox.md#wasmedgeruntime-logos
+          - item:
```
Comment (AkihiroSuda) [inline landscape.yml:1803]: ( I was about to create a new subcategory "Containers for Desktop Operating Systems", but doesn't seem allowed)
https://github.com/cncf/landscape#new-entries )

> We are unlikely to create a new category for products/projects as we'd rather find the best home with the current options.

**Comment by AkihiroSuda** [comment]
CI failure seems unrelated https://app.netlify.com/sites/landscape/deploys/632ae098c96a6a0009851431

```
7:02:45 PM: FATAL: Empty twitter for Serverless Stack: https://twitter.com/ServerlessStack
7:02:45 PM: FATAL: Empty twitter for Tarantool: https://twitter.com/mailru
...
7:02:45 PM: FATAL: Project DevStream has been added to sandbox category but does not belong to CNCF
7:02:45 PM: FATAL: Project KubeArmor has been added to sandbox category but does not belong to CNCF
7:02:45 PM: FATAL: Project Teller has been added to sandbox category but does not belong to CNCF
7:02:45 PM: FATAL: Project wasmCloud has been added to sandbox category but does not belong to CNCF
7:02:45 PM: FATAL: Project OpenELB has been added to sandbox category but does not belong to CNCF
7:02:45 PM: FATAL: Project Nocalhost has been added to sandbox category but does not belong to CNCF
7:02:45 PM: FATAL: Project OpenFunction has been added to sandbox category but does not belong to CNCF
```

**Comment by AndreyKozlov1984** [comment]
The new PR has been created

**Pull request closed without merging**